making-demos.md

```diff
@@ -171,6 +171,27 @@ TODO
 
 TODO
 
+### Prepending the Firefox DevTools intro video
+
+It's a good idea to also prepend the Firefox DevTools intro video in your new video so it looks more official.
+
+You can download this video [on GitHub's LFS storage here](https://github.com/devtools-html/demos/blob/master/assets/bumper%201080.mov).
+
+You can see an example of it being used at the start of [this video](https://www.youtube.com/watch?v=uZQTEPGdjAg).
+
+#### Mac OS
```

Comment:
Pickiness to the max: it's now called macOS as seen in their documents https://www.apple.com/macos/
Not a terrible issue but somehow it stood up in my eyes